features/base/test/test_sudo.py

```diff
@@ -0,0 +1,26 @@
+from helper.utils import read_file_remote as read
+from helper.utils import execute_remote_command as run
+import pytest
+
+
+@pytest.mark.security_id(165)
+def test_sudoers(client, non_container):
+    """
+    Validate that we have the necessary %sudo configration entry in sudoers.
+    Note: we do not use wheel, but instead the sudo group.
+    """
+    # Ensure we have sudo permissions.
+    client._default_to_sudo = True
+    output = run(client, cmd="getent group sudo")
+    assert output.split(":")[0] == 'sudo', "Group 'sudo' is missing!"
+
+    sudoers = read(client,
+                   file="/etc/sudoers",
```

Comment:
Do we want to check /etc/sudoers.d/* as well?

Reply:
may check if we have `@includedir`, `#include` or `#includedir` (see [man of sudoers](https://www.man7.org/linux/man-pages/man5/sudoers.5.html) chapter Including other files from within sudoers)